sgxs-loaders/src/lib.rs

```diff
@@ -14,5 +14,44 @@ extern crate sgx_isa as abi;
 extern crate sgxs as sgxs_crate;
 #[macro_use]
 extern crate failure_derive;
+#[macro_use]
+extern crate bitflags;
 
+mod generic;
 pub mod isgx;
+pub mod sgx_enclave_common;
+
+use std::fmt::Debug;
+use std::os::raw::c_void;
+use std::sync::Arc;
+
+use sgxs_crate::loader;
+
+#[derive(Debug)]
+pub struct Tcs {
+    _mapping: Arc<Debug + Sync + Send>,
```

Comment:
Nice. I suppose this would just be `Arc<Drop + Sync + Send>` if we didn't want `Debug`.